### PR TITLE
Github pages

### PR DESCRIPTION
**Pull request by worldtiki**
Github pages.
Deleted some of the stuff in the README and added a reference to the docs in Github pages.

### CODE REVIEW

**Comment by nikos912000** [comment]
Better to have a separate branch with only the GitHub Pages files and deploy that branch.
This way we'll have a separation between the main project's files and any files related to GH Pages.

That is, can we open a PR with any files needed for GitHub Pages vs a Pitchfork branch (e.g. `gh-pages`). See [here](https://github.com/ExpediaGroup/stream-registry/tree/gh-pages) for an example.

**Comment by nikos912000** [comment]
I believe you need the `Gemfile` to be able to build the GH Pages. Also see [here]
(https://help.github.com/en/articles/setting-up-your-github-pages-site-locally-with-jekyll#step-2-install-jekyll-using-bundler). 

After this addition it looks OK with the following commands locally:
`buindle install`
` bundle exec jekyll serve`

**Comment by worldtiki** [comment]
> I believe you need the `Gemfile` to be able to build the GH Pages. Also see [here](https://help.github.com/en/articles/setting-up-your-github-pages-site-locally-with-jekyll#step-2-install-jekyll-using-bundler).
> 
> After this addition it looks OK with the following commands locally:
> `buindle install`
> ` bundle exec jekyll serve`

Works fine without it: https://worldtiki.github.io/pitchfork/ :p

File: README.md [hunk 1]
```diff
@@ -166,9 +88,10 @@ None of these are essential for a pull request to be approved but they all help.
...
 ## References
+* [Pitchfork Documentation](https://hotelsdotcom.github.io/pitchfork/)
 * [Pitchfork at Docker Hub](https://hub.docker.com/r/hotelsdotcom/pitchfork/)
-* [Haystack](https://github.com/ExpediaDotCom/haystack)
-* [Zipkin](https://github.com/openzipkin/zipkin)
+* [Haystack](https://github.com/ExpediaDotCom/haystack/)
+* [Zipkin](https://github.com/openzipkin/zipkin/)
 
 ## License
 This project is licensed under the Apache License v2.0 - see the LICENSE.txt file for details.
```
[the remark this answers is not in the document]
Comment (worldtiki) [reply README.md]: Sure

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+_Note: When using Docker it is preferable to use the posix notation for the properties (eg. FOO_BAR instead of foo.bar)._
+
+You can run Pitchfork as a Docker container. You can find all available tags in [Docker Hub](https://hub.docker.com/r/hotelsdotcom/pitchfork/).
+
+```bash
+docker run -p 9411:9411 hotelsdotcom/pitchfork:latest
+```
+
+You can override the default properties by with environment variables, for example:
```
[the remark this answers is not in the document]
Comment (worldtiki) [reply docs/index.md]: Ooops :x

**Comment by worldtiki** [comment]
> Better to have a separate branch with only the GitHub Pages files and deploy that branch.
> This way we'll have a separation between the main project's files and any files related to GH Pages.
> 
> That is, can we open a PR with any files needed for GitHub Pages vs a Pitchfork branch (e.g. `gh-pages`). See [here](https://github.com/ExpediaGroup/stream-registry/tree/gh-pages) for an example.

I started with that approach in my fork and thought the extra level of indirection wasn't helpful, that's when I moved to having a docs dir. I had to keep switching between branches to check where/how a property was named/used.

I don't have a strong opinion on this though, I guess both options have pros and cons. I'm happy to move this to if you think it's a good idea. Let me know what you think

**Comment by nikos912000** [comment]
> > I believe you need the `Gemfile` to be able to build the GH Pages. Also see [here](https://help.github.com/en/articles/setting-up-your-github-pages-site-locally-with-jekyll#step-2-install-jekyll-using-bundler).
> > After this addition it looks OK with the following commands locally:
> > `buindle install`
> > ` bundle exec jekyll serve`
> 
> Works fine without it: https://worldtiki.github.io/pitchfork/ :p

Does it work locally? If yes then that's fine.

**Comment by nikos912000** [comment]
> > Better to have a separate branch with only the GitHub Pages files and deploy that branch.
> > This way we'll have a separation between the main project's files and any files related to GH Pages.
> > That is, can we open a PR with any files needed for GitHub Pages vs a Pitchfork branch (e.g. `gh-pages`). See [here](https://github.com/ExpediaGroup/stream-registry/tree/gh-pages) for an example.
> 
> I started with that approach in my fork and thought the extra level of indirection wasn't helpful, that's when I moved to having a docs dir. I had to keep switching between branches to check where/how a property was named/used.
> 
> I don't have a strong opinion on this though, I guess both options have pros and cons. I'm happy to move this to if you think it's a good idea. Let me know what you think

Yep, I see your point. We can keep it under `/docs.

**Pull request merged**